src/assets/i18n/en.json

```diff
@@ -167,7 +167,10 @@
                 "TITLE": "About"
             },
             "SEARCH": {
-                "TITLE": "Search Results",
+              "FORMS": {
+                "ALL": "ALL"
```

Comment:
can we change from all to deafult?

Reply:
and maybe we should not show the possibility to choose a form if there is only 1

Reply:
@eromano I would like to address the selection comment in the upcoming PR. (it requires some changes in adf side)